apps/admin-gui/src/app/users/pages/user-detail-page/user-assignments/user-assignments.component.html

```diff
@@ -0,0 +1,62 @@
+<h1 class="page-title">{{'USER_DETAIL.ASSIGNMENTS.TITLE' | translate}}</h1>
+<mat-spinner class="ms-auto me-auto" *ngIf="initLoading"></mat-spinner>
```

Comment:
Prefer non blocking loading "indication", use template and `*perunWebAppsLoader` directive.

Reply:
There is no need to define multiple templates, if the indicator is the same. Also I think we only need this for the resource table ? The attribute component handles loading for itself. One important note when using the directive parent must have position relative, so the indicator is contained within component.

Reply:
@xflord

Reply:
Sorry, didn't notice this comment. I believe it's better to have the spinner display for every tab, since the attribute components start loading only after the facility loads (so should the user switch facility via the select and that call was slow, no loading indicator would be shown). It looks better now that I used only one template before the tab content, thanks. Also made the same changes for the accounts page (the improvements are more visible there if you want to check)

Reply:
You are right I did not think about that case. That being said right know if you switch facility while on the attributes tab it is not reloaded. Also with the loading changes on accounts page when initializing member is null error is present.

One other thing I still think we don't need spinner1/2/3/. The purpose of template is that it can be reused. You can just define one at the beginning/end of the file and use it as many times as you want. Or I am missing something here ?

Reply:
You are right, I didn't quite understand how templates work. Also I assumed the loader directive worked similarly to ngIf but that's not the case, hence why the attributes didn't reload. Reverted to old spinners for attributes (and whole accounts page)